catalyst_voices/scripts/version_web_assets/web_asset_versioner.dart

```diff
@@ -12,8 +12,8 @@ class WebAssetVersioner {
   static const manuallyVersionedFiles = [
     'sqlite3.wasm',
     'drift_worker.js',
-    'assets/packages/catalyst_key_derivation/assets/js/catalyst_key_derivation_bg.wasm',
-    'assets/packages/catalyst_key_derivation/assets/js/catalyst_key_derivation.js',
+    'assets/packages/catalyst_key_derivation/assets/js/*/catalyst_key_derivation_bg.wasm',
+    'assets/packages/catalyst_key_derivation/assets/js/*/catalyst_key_derivation.js',
     'assets/packages/catalyst_compression/assets/js/catalyst_compression_bg.wasm',
     'assets/packages/catalyst_compression/assets/js/catalyst_compression.js',
```

Comment:
The same for these files as you did for key derivation

```suggestion
'assets/packages/catalyst_compression/assets/js/*/catalyst_compression_bg.wasm',
'assets/packages/catalyst_compression/assets/js/*/catalyst_compression.js',
```

Reply:
Updated to be unified